pkg/cache/context.go

```diff
@@ -660,11 +660,8 @@ func (ctx *Context) RemoveApplication(appID string) error {
 func (ctx *Context) RemoveApplicationInternal(appID string) error {
 	ctx.lock.Lock()
 	defer ctx.lock.Unlock()
-	if _, exist := ctx.applications[appID]; exist {
-		delete(ctx.applications, appID)
-		return nil
-	}
-	return fmt.Errorf("application %s is not found in the context", appID)
+	delete(ctx.applications, appID)
```

Comment:
I think we can treat the message that shows the application doesn't exist as a log information.
Like you did in RemoveTask function.
`
if err != nil {
  log.Logger().Info(" not found, it has been deleted",
  zap.String("Application", appID))
  }
`

Reply:
Hi @wilfred-s , should we keep the message as a log to remind user or just skip the the massage?

Reply:
I would remove the whole return value from the call. We do not return errors as part of a `deleteNode` in the context as an example.

We can log a message, however we should not log it at a INFO level. DEBUG would be more appropriate for the log level. The message should then just say that we're removing the application or the task. Independent of the fact that the delete really happened. Again check `deleteNode` changes in YUNIKORN-1100 (#375)

Reply:
`RemoveApplicationInternal` `RemoveTask` and `removeTask` all should not return an error. Just be silent and execute the remove

Reply:
Thanks @wilfred-s, i got it.
Hi @youxuan0714,   please move the message to debug level and change these functions with no error return.
You can review similar modification in YUNIKORN-1100(#375).

Reply:
Ok, I will try again, thanks.